backend/src/v2/driver/driver.go

```diff
@@ -320,9 +321,80 @@ func initPodSpecPatch(
 
 	addModelcarsToPodSpec(executorInput.GetInputs().GetArtifacts(), userEnvVar, podSpec)
 
+	if needsWorkspaceMount(executorInput) {
+		// Validate that no user volume mounts conflict with the workspace
+		if err := validateVolumeMounts(podSpec); err != nil {
+			return nil, fmt.Errorf("failed to validate volume mounts: %w", err)
+		}
+
+		// Uses Argo template variable to reference the PVC created by volumeClaimTemplates
+		// Argo resolves {{workflow.name}}-kfp-workspace to the actual PVC name at runtime
+		pvcName := "{{workflow.name}}-" + component.WorkspaceVolumeName
```

Comment:
Could you please update the design document to match the implementation for the PVC name? For example:

```diff
diff --git a/proposals/11875-pipeline-workspace/README.md b/proposals/11875-pipeline-workspace/README.md
index 1c355aabc..41803b4ef 100644
--- a/proposals/11875-pipeline-workspace/README.md
+++ b/proposals/11875-pipeline-workspace/README.md
@@ -509,7 +509,7 @@ kind: Workflow
 spec:
   volumeClaimTemplates:
     - metadata:
-        name: kfp-workspace-46f1d52e-c72b-42fc-88ae-789edf7c33fd # The suffix is the run ID
+        name: kfp-workspace
       spec:
         accessModes:
           - ReadWriteMany
```